README.md

```diff
@@ -58,6 +58,18 @@ Allow you to enable/disable the anchor link in the headings
 
 Allow you to customize the anchor link symbol
 
+#### `anchorLinkSpace`
+
+(default: `true`)
+
+Allow you to enable/disable inserting a space between the anchor link and heading.
+
+#### `anchorLinkSymbolClassName`
+
+(default: `null`)
+
+Allow you to customize the anchor link symbol class name. If not null, symbol will be rendered as `<span class="anchorLinkSymbolClassName">anchorLinkSymbol</span>`.
```

Comment:
It will be empty if you set `anchorLinkSymbol: ''`. Just for more flexibility. User can decide empty or not.